.github/workflows/woodwork_main_performance_tests.yaml

```diff
@@ -0,0 +1,143 @@
+name: Run looking glass performance tests
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  performance_tests:
+    name: Run looking glass performance tests
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+    env:
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_SECRET_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      AWS_DEFAULT_REGION: 'us-east-1'
+      AWS_SESSION_TOKEN: ""
+    steps:
+      - name: Checkout woodwork
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - name: Get commit hashes
+        id: get_hashes
+        run: |
+          current_hash=$(git rev-parse --short HEAD)
+          echo "Latest commit hash: $current_hash"
+          echo "::set-output name=current_hash::$current_hash"
+          previous_hash=$(git rev-parse --short HEAD~1)
+          echo "Previous commit hash: $previous_hash"
+          echo "::set-output name=previous_hash::$previous_hash"
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
+      - name: Checkout looking glass
+        uses: actions/checkout@v2
+        with:
+          repository: alteryx/looking-glass
+          ref: master
+          token: ${{ secrets.LOOKING_GLASS_TOKEN }}
+      - name: Installing Dependencies
+        run: |
+          make installdeps
+          make installdeps-test
+      - name: Run performance test (current)
+        id: current
+        run: |
+          response=$( looking-glass run-woodwork --username machineFL --scenarios-yaml woodwork_scenarios.yaml --woodwork-branch ${{ steps.get_hashes.outputs.current_hash }} --job-desc ${{ steps.get_hashes.outputs.current_hash }} )
+          echo "::set-output name=job_id::$(echo $response | sed 's/.*Job ID: \([^ ]*\).*/\1/')"
```

Comment:
What's going on here in the `echo`?

Reply:
This echo moves the results of the job into an environment variable named after the job_id